cognite/neat/_data_model/models/dms/_constraints.py

```diff
@@ -16,7 +17,7 @@ class UniquenessConstraintDefinition(ConstraintDefinition):
     properties: list[str] = Field(
         description="List of properties included in the constraint.", min_length=1, max_length=10
     )
-    by_space: bool | None = Field(default=None, description="Whether to make the constraint space-specific.")
+    by_space: Bool | None = Field(default=None, description="Whether to make the constraint space-specific.")
```

Comment:
In the table format of Neat we need to handle conversion of string like  'True' and 'false' to bool.